src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -163,47 +148,39 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         overlays['Bike Share Locations'] = overlaysControl.bikeShareOverlay();
         overlays['Bike Routes'] = overlaysControl.bikeRoutesOverlay(map);
 
-        // TODO: handle hiding layers on home view more cleanly
-        // when user switches back to home view from map view
-        // Would be better to initialize but not add to map, to avoid flashing,
-        // although it probably won't be visible due to centering of home page text.
-        if (homepage) {
-            overlays['Bike Share Locations'].eachLayer(function(layer) { layer.hide(); });
-            overlays['Bike Routes'].eachLayer(function(layer) { layer.hide(); });
-        }
-
         // TODO: re-enable when Uwishunu feed returns
         //overlays['Nearby Events'] = overlaysControl.nearbyEventsOverlay();
         //overlays['Nearby Events'].addTo(map);
     }
 
     function initializeLayerControl() {
-        layerControl = cartodb.L.control.layers(basemaps, overlays, {
-            position: 'bottomright',
-            collapsed: false
-        });
+        if (!layerControl) {
```

Comment:
Noting here that the changes for this section will conflict with #625 so we'll need to fix that up